commands/operator-sdk/cmd/scorecard/basic_tests.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
```

Comment:
Leave a new line. 
Or run `goimports` to organize the imports by groups. I think we've been following that convention more closely in recent PRs.
https://github.com/golang/go/wiki/CodeReviewComments#imports